### PR TITLE
feature: dependency overhaul

### PR DESCRIPTION
**Pull request by comountainclimber**
**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

File: config/webpack.config.prod.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ const path = require('path')
 const webpack = require('webpack')
 const HtmlWebpackPlugin = require('html-webpack-plugin')
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
-const ExtractTextPlugin = require('extract-text-webpack-plugin')
```
Comment (comountainclimber) [inline config/webpack.config.prod.js:5]: `ExtractTextPlugin` is deprecated and incompatible with node `16.14.2`

File: package.json [hunk 2]
```diff
@@ -189,13 +191,13 @@
     "babel-preset-react": "6.24.1",
     "babel-preset-stage-0": "6.24.1",
     "cross-env": "5.1.1",
-    "css-loader": "0.28.7",
-    "electron": "^10.0.0",
-    "electron-builder": "22.11.11",
+    "css-loader": "4.2.0",
+    "electron": "19.0.8",
```
Comment (comountainclimber) [inline package.json:196]: 👏

File: package.json [hunk 3]
```diff
@@ -222,15 +224,14 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "22.4.2",
     "json-loader": "0.5.7",
-    "node-sass": "4.13.1",
```
Comment (comountainclimber) [inline package.json:225]: `node-sass` is another deprecated project incompatible with electron 19 and node 16

File: app/core/storage.js [hunk 4]
```diff
@@ -1,9 +1,20 @@
+import { ipcRenderer } from 'electron'
```
Comment (comountainclimber) [inline app/core/storage.js:1]: The storage module needed to be refactored for some reason the default path it was using changed during the electron update. Its working exactly as expected with these changes

**Review by melanke — APPROVED** [review]

File: main.js [hunk 5]
```diff
@@ -85,6 +85,7 @@ app.on('ready', () => {
       contextIsolation: true,
       webPreferences: {
         enableRemoteModule: true,
+        contextIsolation: false,
```
Comment (comountainclimber) [inline main.js:88]: This was already turned off in our previous builds but the setting has changed to defaulting to true in new versions of electron thus requiring this explicit flag

**Pull request merged**